Description: Switch build-dep from libappindicator3-dev to libayatana-appindicator3-dev.
Author: Francisco Vilmar Cardoso Ruviaro <francisco.ruviaro@riseup.net>
Last-Update: 2021-10-19

--- whatsapp-for-linux-1.3.0.orig/CMakeLists.txt
+++ whatsapp-for-linux-1.3.0/CMakeLists.txt
@@ -19,7 +19,7 @@ add_custom_command(
 FIND_PACKAGE(PkgConfig REQUIRED)
 PKG_CHECK_MODULES(GTKMM3 gtkmm-3.0 REQUIRED)
 PKG_CHECK_MODULES(WEBKIT webkit2gtk-4.0 REQUIRED)
-PKG_CHECK_MODULES(APP_INDICATOR appindicator3-0.1 REQUIRED)
+PKG_CHECK_MODULES(APP_INDICATOR ayatana-appindicator3-0.1 REQUIRED)
 include_directories(${GTKMM3_INCLUDE_DIRS} ${WEBKIT_INCLUDE_DIRS} ${APP_INDICATOR_INCLUDE_DIRS})
 
 add_executable(whatsapp-for-linux src/main.cpp src/Version.hpp src/Resources.c src/Application.cpp src/MainWindow.cpp src/PhoneNumberDialog.cpp src/TrayIcon.cpp src/WebView.cpp src/Settings.cpp)
--- whatsapp-for-linux-1.3.0.orig/src/TrayIcon.hpp
+++ whatsapp-for-linux-1.3.0/src/TrayIcon.hpp
@@ -1,6 +1,6 @@
 #pragma once
 
-#include <libappindicator/app-indicator.h>
+#include <libayatana-appindicator/app-indicator.h>
 #include <gtkmm/menu.h>
 
 class TrayIcon
